doc/modules/covariance.rst

```diff
@@ -42,8 +48,6 @@ by the user, and ``assume_centered=True`` should be used.
 
 .. rubric:: Examples
 
```

Comment:
```suggestion
```
Remove section title including blank line.

The same applies below.

Reply:
I'll do it right away. This is good to know for future PRs as well. Thanks

Reply:
So, when you go into the CI checks overview below, you find one that says "Check the rendered docs here!". In there, you can look into how the CI build the documentation for the changed files.

For instance: [modules/generated/sklearn.covariance.OAS.html](https://output.circle-artifacts.com/output/job/fe3895c3-b800-468d-b0ec-12909ac0be0f/artifacts/0/doc/modules/generated/sklearn.covariance.OAS.html) [[dev](https://scikit-learn.org/dev/modules/generated/sklearn.covariance.OAS.html), [stable](https://scikit-learn.org/stable/modules/generated/sklearn.covariance.OAS.html)]

Where the first link is your branches version, the second is the main dev branch and the third link is the last releases scikit-learn version you can find on the website.

It helps to check if everything went well.